smoketest/smoketest.go

```diff
@@ -32,27 +33,29 @@ func HandleSmokeTest(ctx context.Context, opts Options) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		b, err := io.ReadAll(r.Body)
 		if err != nil {
-			logs.Error(errors.New("reading body failed").Wrap(err))
-			w.WriteHeader(http.StatusInternalServerError)
+			httpcmn.InternalServerError(w, errors.New("reading body failed").Wrap(err))
 			return
 		}
 
 		var req hsmoketest.SmokeTestRequest
 		if err := json.Unmarshal(b, &req); err != nil {
-			logs.Error(errors.New("marshaling request failed").Wrap(err))
-			w.WriteHeader(http.StatusInternalServerError)
+			httpcmn.BadRequest(w, httpcmn.ErrBadRequest)
```

Comment:
Please send a response like "request json is invalid".

Reply:
`httpcmn.ErrBadRequest` will send a response with `Invalid request body`
https://github.com/aukilabs/hagall-common/blob/e1d046a9b16cec0d6ce5203b540606643e1603fc/http/error.go#L20